cargo/cargo_build_script_runner/bin.rs

```diff
@@ -53,6 +53,30 @@ fn run_buildrs() -> Result<(), String> {
     create_dir_all(&out_dir_abs)
         .unwrap_or_else(|_| panic!("Failed to make output directory: {:?}", out_dir_abs));
 
+    // Symlink the execroot to the manifest_dir so that we can use relative paths in the arguments.
+    let exec_root_paths = std::fs::read_dir(&exec_root)
+        .map_err(|err| format!("Failed while listing exec root: {err:?}"))?;
+    for path in exec_root_paths {
+        let path = path
+            .map_err(|err| format!("Failed while getting path from exec root listing: {err:?}"))?
+            .path();
+
+        let file_name = path
+            .file_name()
+            .ok_or_else(|| "Failed while getting file name".to_string())?
+            .to_str()
+            .ok_or_else(|| "Failed while converting file name to string".to_string())?;
```

Comment:
```suggestion
            .ok_or_else(|| "Failed while getting file name".to_string())?;
```

You can use an `OsString` to `Path::join`